keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,569 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""EfficientNet V2 models for KerasCV.
+
+Reference:
+    - [EfficientNetV2: Smaller Models and Faster Training](https://arxiv.org/abs/2104.00298) (ICML 2021)
+    - [Based on the original keras.applications EfficientNetV2](https://github.com/keras-team/keras/blob/master/keras/applications/efficientnet_v2.py)
+"""  # noqa: E501
+
+import copy
+import math
+
+from tensorflow import keras
+from tensorflow.keras import layers
+
+from keras_cv.layers import FusedMBConvBlock
+from keras_cv.layers import MBConvBlock
+from keras_cv.models import utils
+from keras_cv.models.backbones.backbone import Backbone
+from keras_cv.models.backbones.efficientnet_v2.block_args import (
+    DEFAULT_BLOCKS_ARGS,
+)
+from keras_cv.models.backbones.efficientnet_v2.efficientnet_v2_backbone_presets import (  # noqa: E501
+    backbone_presets,
+)
+from keras_cv.models.backbones.efficientnet_v2.efficientnet_v2_backbone_presets import (  # noqa: E501
+    backbone_presets_with_weights,
+)
+from keras_cv.utils.python_utils import classproperty
+
+CONV_KERNEL_INITIALIZER = {
```

Comment:
Let's make these inlined classes rather than serialized objects. See [example](https://github.com/keras-team/keras-nlp/blob/edd6b6ba58afde819e7b07d4a0aa47f15a8a996b/keras_nlp/models/bert/bert_backbone.py#L30)

Reply:
Done